src/datadoc/state.py

```diff
@@ -20,3 +23,7 @@
 metadata: DataDocMetadata
 
 current_metadata_language: SupportedLanguages
+
+SOURCE_URL = "https://www.ssb.no/xp/_/service/mimir/subjectStructurStatistics"
```

Comment:
We shouldn't hard code this, but instead get it from the config, we added functionality for that in #158